tests/foreman/api/test_contentmanagement.py

```diff
@@ -493,28 +491,27 @@ def test_positive_capsule_sync(self):
         sync_status = capsule.content_get_sync()
         last_sync_time = sync_status['last_sync_time']
 
-        # If BZ1439691 is open, need to sync repo once more, as repodata
-        # will change on second attempt even with no changes in repo
-        if bz_bug_is_open(1439691):
-            repo.sync()
-            repo = repo.read()
-            cv.publish()
-            cv = cv.read()
-            self.assertEqual(len(cv.version), 2)
-            cv.version.sort(key=lambda version: version.id)
-            cvv = cv.version[-1].read()
-            promote(cvv, lce.id)
-            cvv = cvv.read()
-            self.assertEqual(len(cvv.environment), 2)
-            sync_status = capsule.content_get_sync()
-            self.assertTrue(
-                len(sync_status['active_sync_tasks']) >= 1
-                or sync_status['last_sync_time'] != last_sync_time
-            )
-            for task in sync_status['active_sync_tasks']:
-                entities.ForemanTask(id=task['id']).poll()
-            sync_status = capsule.content_get_sync()
-            last_sync_time = sync_status['last_sync_time']
+        # If BZ1439691 is CLOSED WONTFIX, need to sync repo once more,
```

Comment:
@mirzal another BZ can be opened as a Duplicate of this one, which will make it dependent of this one and if the dup is resolved this one whould also be considered resolved. Our nagger script will need to lookup for duplicates and other relations.

Reply:
I wonder if there is any instance of bug being closed WONTFIX, then getting a duplicate, and then that duplicate getting properly fixed. I would expect CLOSED WONTFIX duplicates to be closed almost automatically.

Anyway, I was mostly referring to grammar of this comment. "If BZ1439691 is CLOSED WONTFIX" is misleading for me, because that BZ is CLOSED WONTFIX already. If we care about possible duplicates with other states, I propose "If BZ1439691 and all duplicates are CLOSED WONTFIX".